meedan/check-web
# Handle unique key constraint error in feed invitation response

## Description

This commit fixes the scenario where:
Two users in the same workspace could receive invitations to join a shared feed. If the first one accepts, and then the second one tries to accept shortly after, then there is currently an error message about a unique key constraint conflict from the database.

Reference: CV2-4038

## Type of change


## How has this been tested?


Steps to Test:
User A and User B are in Workspace A.
User C invites User A and User B to join a shared feed.
User A and User B navigate to the page to accept the invitation.
User A accepts the invitation.
Verify that no error message is displayed and that the invitation is successfully accepted.
User B attempts to accept the invitation shortly after User A.
Verify that no error message related to a unique key constraint conflict from the database appears.
Verify that User B is redirected to the feed page 

## Checklist
